packages/blade/src/components/Alert/StyledAlert.web.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components';
+
+import type { StyledAlertProps } from './StyledAlert.d';
+import { makeSpace } from '~utils';
+
+const StyledAlert = styled.div<StyledAlertProps>((props) => {
+  const { theme, contrastType, intent, isFullWidth, isBorderless } = props;
+  const _isFullWidth = isFullWidth || isBorderless; // a borderless alert is automatically set to full width
+  const feedbackColors = theme.colors.feedback;
+
+  return {
+    background: feedbackColors.background[intent][contrastType],
+    padding: `${makeSpace(theme.spacing[3])} ${makeSpace(theme.spacing[4])}`,
+    borderRadius: isBorderless ? theme.border.radius.none : theme.border.radius.medium,
+    borderColor: feedbackColors.border[intent][contrastType],
+    borderWidth: isBorderless ? theme.border.width.none : theme.border.width.thin,
```

Comment:
This isn't needed, is it? We're using object styles form. Styled components adds `px`, etc.

Reply:
Personally, I think for time it makes sense but probably an overkill for things which are already handled by the platform / styled-components like pixel size 😄 

Also I feel we don't know future problems right now to confirm if we'll *ever* need to change the decision later. If we do, it might anyway need a larger refactor. Though for consistency sake, can definitely refactor this in.

Reply:
Updated in https://github.com/razorpay/blade/pull/666